Content.Server/ADT/GameTicking/Rules/Components/ShipsVsShipsRuleComponent.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Robust.Shared.Player;
+
+namespace Content.Server.ADT.GameTicking.Rules;
+
+[RegisterComponent]
+public sealed partial class ShipsVsShipsRuleComponent : Component
+{
+    // Тип победы в игре (по умолчанию - нейтральный)
```

Comment:
```suggestion
[RegisterComponent] // Комментарий тест
public sealed partial class ShipsVsShipsRuleComponent : Component
{
    // Тип победы в игре (по умолчанию - нейтральный)
```

давай добавим тут коммент?